PROTO_tests/tests/alignment_test.py

```diff
@@ -0,0 +1,142 @@
+import pytest
+import json
+import arkouda as ak
+
+DATA_TYPES = [ak.int64, ak.uint64, ak.float64]
+class TestAlignment:
```

Comment:
we need to get u set up with `black` it's our autoformatter. We run it on files to make sure we have a consistent formatting that follow best practices across all files. It can be integrated into pycharm as an external tool or used directly from the command line, [here's](https://bears-r-us.github.io/arkouda/CONTRIBUTING_LINK.html#python3) our docs on it

This popped out to me because according to [pep-8](https://pep8.org/#blank-lines) you're supposed to 
> Surround top-level function and class definitions with two blank lines.

Reply:
Noted, will start using it.